README.md

```diff
@@ -22,6 +22,9 @@ COMMANDS:
 GLOBAL OPTIONS:
    --scope value  Scope of command. For example: event, build, pipeline
    --type value   Type of the command. For example: cache, artifacts, steps (default: "stable")
+   --upload-timeout value    Specifies the upload timeout in seconds. (default: 60) [$SD_STORE_CLI_UPLOAD_HTTP_TIMEOUT]
```

Comment:
I think the command-line specification would be tidier if we consolidate the timeout option in the same way as the scope and type options, and reflect it in the get, set, and remove commands.

Reply:
Thank you for your comment.
I know what you are saying. May be I should try that , but I don't think make a big refactoring. 
I will try to rewrite a bit .

Reply:
Fixed. Please review the changes I fixed whether it match your vision or not.

If that do match your vision , I will add more test cases.

Reply:
👍